src/app/shared/components/login-page/login-page.component.ts

```diff
@@ -25,6 +25,7 @@ export class LoginPageComponent implements OnInit {
   }
 
   ngOnInit() {
+    this.loginService.cleanCache();
```

Comment:
maybe we should move it to "after the user has a successful login ". like after resume session